Source/Examples/ExampleLibrary/Series/AreaSeriesExamples.cs

```diff
@@ -32,7 +32,20 @@ public static PlotModel DifferentColors()
             return plotModel1;
         }
 
-        [Example("Crossing lines")]
+		[Example("Two colors mode")]
```

Comment:
Thanks for the example! But see my comment on #819. This can be done by two `AreaSeries`. It would be good to have an example that cannot be created by two `AreaSeries`.